src/platform_impl/windows/window.rs

```diff
@@ -472,27 +472,39 @@ impl Window {
     }
 
     unsafe fn handle_os_dragging(&self, wparam: WPARAM) {
-        let points = {
-            let mut pos = unsafe { mem::zeroed() };
-            unsafe { GetCursorPos(&mut pos) };
-            pos
-        };
-        let points = POINTS {
-            x: points.x as i16,
-            y: points.y as i16,
-        };
-        unsafe { ReleaseCapture() };
-
         self.window_state_lock().dragging = true;
 
-        unsafe {
-            PostMessageW(
-                self.hwnd(),
-                WM_NCLBUTTONDOWN,
-                wparam,
-                &points as *const _ as LPARAM,
-            )
-        };
+        let (tx, rx) = channel::<()>();
```

Comment:
Do we really need to do a `roundtrip`?

Reply:
The function was blocking before, I kept it as-is, seemed sensible.

Reply:
yeah, it's just other functions are non-blocking doing the same things, in general, I think windows backend lacks the policy when doing so, so I'd prefer to either we change all functions to be blocking or we just use what we have now and don't block.

Wayland for example is also non-blocking.